plugin/trino-hudi/pom.xml

```diff
@@ -424,6 +346,83 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-common</artifactId>
+            <version>${dep.hudi.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
```

Comment:
I checked `mvn dependency:tree -pl :trino-hudi` and it appears that we are excluding all of the dependencies. If that is still the case, change this to
```xml
<exclusion>
    <groupId>*</groupId>
    <artifactId>*</artifactId>
</exclusion>